core/src/main/java/org/mule/runtime/core/internal/policy/DefaultPolicyInstance.java

```diff
@@ -214,9 +218,9 @@ public ProcessingStrategy getProcessingStrategy() {
     private static Logger LOGGER = getLogger(PolicyProcessingStrategy.class);
     private static int SCHEDULER_BUSY_RETRY_INTERVAL_MS = 2;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "SCHEDULER_BUSY_RETRY_INTERVAL_MS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S3008](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3008))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7629&issues=AWiGlnXCotIXnmsUoniV&open=AWiGlnXCotIXnmsUoniV)

[ISSUE_KEY]: AWiGlnXCotIXnmsUoniV